chia/wallet/puzzles/pool_waitingroom_innerpuz.clvm

```diff
@@ -49,14 +50,14 @@
 
   (if spend_type
     (list (list ASSERT_HEIGHT_RELATIVE RELATIVE_LOCK_HEIGHT)
-          (list CREATE_COIN destination_puzhash my_amount)
-          (list AGG_SIG_ME OWNER_PUBKEY (sha256tree (list destination_puzhash my_amount key_value_list) ()))
+          (list CREATE_COIN p2 my_amount)
```

Comment:
Consider refactoring this case into an inline function so you can relabel `p2` to a name that makes it more clear what it represents (like `destination_puzzle_hash`).